src/Sylius/Bundle/AdminBundle/package.json

```diff
@@ -8,7 +8,7 @@
     "@popperjs/core": "^2.11.8",
     "@symfony/stimulus-bridge": "^3.2.2",
     "@symfony/webpack-encore": "^3.1.0",
-    "@tabler/core": "tabler/tabler#dev",
+    "@tabler/core": "^1.0.0-beta20",
```

Comment:
Just in case, you sure about this change? It shouldn't be related to current PR.

Reply:
this has been fixed on 2.0 branch, now the upmerge is configured, but it was not.